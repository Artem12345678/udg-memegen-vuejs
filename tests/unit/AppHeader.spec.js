import { shallowMount, createLocalVue } from '@vue/test-utils';
import { BootstrapVue } from 'bootstrap-vue';

import AppHeader from '@/components/AppHeader/AppHeader.vue';

const localVue = createLocalVue();

localVue.use(BootstrapVue);

describe('AppHeader.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(AppHeader, {
      localVue,
    });
  });

  it('matches snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
