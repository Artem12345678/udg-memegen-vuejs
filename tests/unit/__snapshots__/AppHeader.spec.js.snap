// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader.vue matches snapshot 1`] = `
<header
  class="header bg-primary"
>
  <div
    class="container"
  >
    <b-navbar-stub
      tag="nav"
      toggleable="lg"
      type="dark"
    >
      <b-navbar-brand-stub
        class="brand"
        event="click"
        routertag="a"
        tag="div"
        target="_self"
        to="/"
      >
        MemeGen
      </b-navbar-brand-stub>
       
      <b-navbar-toggle-stub
        label="Toggle navigation"
        target="nav-collapse"
      />
       
      <b-collapse-stub
        id="nav-collapse"
        isnav="true"
        tag="div"
      >
        <b-navbar-nav-stub
          class="ml-auto"
          tag="ul"
        >
          <b-nav-item-stub
            event="click"
            routertag="a"
            target="_self"
            to="/"
          >
            Home
          </b-nav-item-stub>
        </b-navbar-nav-stub>
      </b-collapse-stub>
    </b-navbar-stub>
  </div>
</header>
`;
